### ISSUE TITLE
[WOTLK][Grizzly Hills]Venture Bay Light House

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4436]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Venture Bay Light House the little island on Venture Bat. The territory currently to both factions.
**Expected behaviour**: Tell us what should happen instead
The island should only spawn npc's based off of what faction has captured it. 
**Steps to reproduce the problem**:

1. Go to Venture Bay Light House
2. Watch both faction npc's spawn.
3. 

**Include proofs for this behaviour**
![wowscrnshot_021417_175151](https://cloud.githubusercontent.com/assets/12252441/22953229/4a632faa-f2de-11e6-8fa0-8a1636d93285.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
[Venture Bay](http://wowwiki.wikia.com/wiki/Venture_Bay)
The Venture Bay lighthouse is located in the very south end of the bay. It is caught through PvP means similar to the Broken Hill, The Stadium and The Overlook in Hellfire Peninsula. The faction in control is given access to the daily PvP quest Alliance 15 [74 Daily] Riding the Red Rocket/Horde 15 [74 Daily] Riding the Red Rocket and the NPC who trades Venture Coins for gear.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
hm yesh the whole battleground needs to be implemented

**Comment by Rushor** [comment]
Scripts/OutdoorPvP: Implement Venture Bay PvP

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4436
* commit: 09bfdd0328a290d2030d506994563d56559032e9

![GitHub Logo](http://i.imgur.com/mYOrRk0.jpg)
![GitHub Logo](http://i.imgur.com/DX15AtJ.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, this has been a bit tricky as there were multiple bugs with the spawns and faction takeover.

Everything should be fixed now.
